sdk/appcenter/src/test/java/com/microsoft/appcenter/channel/OneCollectorChannelListenerTest.java

```diff
@@ -92,18 +95,18 @@ public void enqueueConvertedLogs() {
         when(originalLog.getTransmissionTargetTokens()).thenReturn(new HashSet<>(Collections.singletonList("t1")));
 
         /* Mock a log. */
-        CommonSchemaLog log1 = mock(CommonSchemaLog.class);
```

Comment:
I know this is not necessarily in scope for this PR, but this test is doing a lot.  I'd like to see it split up.

Reply:
Not sure how to split as we need to test reset the sequence of epoch and seq in the same test.